packages/icons/package.json

```diff
@@ -11,5 +11,9 @@
     "type": "git",
     "url": "https://github.com/lmc-eu/spirit-design-system.git",
     "directory": "packages/icons"
+  },
+  "scripts": {
+    "prebuild": "yarn svg:process",
+    "svg:process": "node ./scripts/svgProcess.js"
```

Comment:
Please add also a `build` script that will copy `package.json` and `README.md` like in https://github.com/lmc-eu/spirit-design-system/blob/main/packages/design-tokens/package.json#L16